core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/JsonCodec.scala

```diff
@@ -134,7 +134,7 @@ object JsonCodec {
   implicit val urlDecoder: Decoder[URL] =
     Decoder.decodeString.emap(s => Either.catchNonFatal(new URL(s)).leftMap(_.getMessage))
   implicit val diskSizeDecoder: Decoder[DiskSize] =
-    Decoder.decodeInt.emap(d => if (d < 0) Left("Negative number is not allowed") else Right(DiskSize(d)))
+    Decoder.decodeInt.emap(d => if (d < 50) Left("Minimum disk size is 50GB.") else Right(DiskSize(d)))
```

Comment:
@Qi77Qi would we want to keep the "Negative number is not allowed" error and add a new one for minimum disk size or replace it like I've done in this PR? looking at fixing unit tests now and want to make sure I don't get rid of functionality that we may still want

Reply:
I think just erroring min disk is 50GB is clear enough...thanks for the quick fix 👍 ...maybe reword this to `Minimum required disk size is 50GB.`?

Just to double check, this is true for GCE VMs runtimes as well? cc @rtitle  for confirmation

Reply:
Sounds good, will update! thank you :)